core/src/main/java/nl/nn/adapterframework/align/Json2Xml.java

```diff
@@ -217,9 +215,9 @@ public Set<String> getAllNodeChildNames(XSElementDeclaration elementDeclaration,
 			JsonObject o = (JsonObject)node;
 			if (o.isEmpty()) {
 				if (log.isTraceEnabled()) log.trace("no children");
-				return new LinkedHashSet<String>();
+				return new LinkedHashSet<>();
```

Comment:
Als het kan, vind ik dit altijd nog wat explicieter:

```suggestion
				return Collections.emptySet();
```

Reply:
Ik zal kijken of die return-value ergens nog gemuteerd kan worden.